docs/scos/dev/glue-api-guides/202307.0/glue-api-guides.md

```diff
@@ -26,3 +26,4 @@ This section contains a collection of guides for working with the Old Spryker Gl
 * [Glue Spryks](/docs/scos/dev/glue-api-guides/{{page.version}}/glue-spryks.html)
 * [Glue infrastructure](/docs/scos/dev/glue-api-guides/{{page.version}}/old-glue-infrastructure/glue-infrastructure.html)
 * [Glue API tutorials](/docs/scos/dev/glue-api-guides/{{page.version}}/glue-api-tutorials/glue-api-tutorials.html)
+* [Use API Key authorization](/docs/scos/dev/glue-api-guides/{{page.version}}/use-api-key-authorization.html)
```

Comment:
As we can see here, it is high-level list of the glue infrastructure links, not a list of specific how-tos, please move this to `docs/scos/dev/glue-api-guides/202204.0/decoupled-glue-infrastructure/authentication-and-authorization.md`